ydb/core/tx/schemeshard/ut_base/ut_base.cpp

```diff
@@ -11790,40 +11790,70 @@ Y_UNIT_TEST_SUITE(TSchemeShardTest) {
         TTestEnv env(runtime);
         ui64 txId = 100;
 
-        TestCreateSubDomain(runtime, ++txId, "/MyRoot", R"(
-            Name: "test"
-        )");
-
         TString newOwner = "user1";
 
+        TApplyIfUnit applyIfUnit;
+        applyIfUnit.AllowedPathTypes = std::vector<NKikimrSchemeOp::EPathType>();
+        applyIfUnit.AllowedPathTypes.value().push_back(NKikimrSchemeOp::EPathTypeSubDomain);
+        applyIfUnit.AllowedPathTypes.value().push_back(NKikimrSchemeOp::EPathTypeExtSubDomain);
```

Comment:
Also, is there a way to support list or aggregate initialization of the applyIf object (`{.PathVersion = 5, .PathTypes = {NKikimrSchemeOp::EPathTypeSubDomain}}`)?